config/tests/samples/create/harness.go

```diff
@@ -305,17 +311,50 @@ func NewHarness(ctx context.Context, t *testing.T) *Harness {
 		kccConfig.HTTPClient = &http.Client{Transport: t}
 	}
 
-	// Intercept (and log) TF requests
-	transport_tpg.DefaultHTTPClientTransformer = func(ctx context.Context, inner *http.Client) *http.Client {
-		ret := inner
-		if t := ctx.Value(httpRoundTripperKey); t != nil {
-			ret = &http.Client{Transport: t.(http.RoundTripper)}
+	if targetGCP := os.Getenv("E2E_GCP_TARGET"); targetGCP == "vcr" {
+		// Initialize VCR for TF requests
+		input := os.Getenv("VCR_MODE")
+		var VCRMode recorder.Mode
+		if input == "record" {
+			VCRMode = recorder.ModeRecordOnly
+		} else if input == "replay" {
+			VCRMode = recorder.ModeReplayOnly
+		} else {
+			t.Fatalf("[VCR] Input mode is not supported.")
```

Comment:
Nit: consider including the value (and I also suggest hinting that the env var VCR_MODE should be set), e.g.

`t.Fatalf("when using VCR mode, VCR_MODE should be set to record or replay; value %q is not known", input)`